discovery-provider/src/queries/get_unpopulated_tracks.py

```diff
@@ -0,0 +1,78 @@
+import json
+from flask.json import dumps
+
+from src.utils import redis_connection
+from src.models import Track
+from src.utils import helpers
+
+ttl_sec = 60
+
+
+def get_track_id_cache_key(id):
+    return "track:id:{}".format(id)
+
+
+def get_cached_tracks(track_ids):
```

Comment:
WDYT about a helper fn for get and set that is shared across both of the `get_unpopulated` methods so we're not copy pasting?